.github/workflows/health-metrics-pull-request.yml

```diff
@@ -42,10 +42,12 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 22.10.0
-      - uses: 'google-github-actions/auth@v0'
+      # This commit represents v0.8.3
+      - uses: 'google-github-actions/auth@c4799db9111fba4461e9f9da8732e5057b394f72'
```

Comment:
Any reason why this one is in quotes?

Reply:
It was originally in quotes, didn't want to change it. I think both syntaxes are okay but the quotes are needed if there are special characters or something, so the original author was probably being careful.